_security-analytics/api-tools/detector-api.md

```diff
@@ -7,60 +7,58 @@ nav_order: 35
+`schedule.period.unit` | String | The interval's unit of time.
+`inputs` | Object | Detector inputs for its definition.
+`inputs.detector_input` | Array | An array that contains the index and definition used to create the detector. For the initial release in 2.4, only one log data source is allowed for the detector. Support for associating multiple log data sources with a single detector will be available in a future release. See description for `inputs.detector_input.indices` and the log data source.
+`inputs.detector_input.description` | String | Description of the detector. Optional.
+`inputs.detector_input.custom_rules` | Array | Detector inputs for custom rules. At least one rule must be specified for a detector. Optional if pre-packaged rules are specified.
+`inputs.detector_input.custom_rules.id` | String | A valid rule ID for the custom rule generated by the user. Valid rules are formatted as a globally, or, universally unique identifier (UUID) See [Universally unique identifier](https://en.wikipedia.org/wiki/Universally_unique_identifier) for more information.
+`inputs.detector_input.indices` | Array | The log data source used for the detector, which can be either an index name or index pattern. Currently, only one entry is supported with plans to support multiple indexes in a future release. Required.
+`inputs.detector_input.pre_packaged_rules` | Array | Detector inputs for pre-packaged rules (as opposed to custom rules). At least one rule must be specified for a detector. Optional if custom rules are specified.
+`inputs.detector_input.pre_packaged_rules.id` | String | The rule ID for pre-packaged rules. See [Search Pre-Packaged Rules]({{site.url}}{{site.baseurl}}/security-analytics/api-tools/rule-api/#search-pre-packaged-rules) for information on how to use the API to search for rules and obtain rule IDs in results.
+`triggers` | Array | Trigger settings for alerts.
+`triggers.ids` | Array | A list of rule IDs that become part of the trigger condition.
+`triggers.tags` | Array | Tags are specified in a security rule. Tags can then be selected and applied to the alert trigger to focus the trigger conditions for alerts. See an example of how tags are used in a Sigma rule in Sigma's [Rule Creation Guide](https://github.com/SigmaHQ/sigma/wiki/Rule-Creation-Guide#tags).
+`triggers.id` | String | The unique ID for the trigger.
+`triggers.sev_levels` | Array | Sigma rule severity levels: `informational`; `low`; `medium`; `high`; `criticial`. See [Level](https://github.com/SigmaHQ/sigma/wiki/Rule-Creation-Guide#level) in the Sigma Rule Creation Guide.
+`triggers.name` | String | The name of the trigger. Name should only consist of upper and lowercase letters, numbers 0-9, hyphens, spaces, and underscores. Use between 5 and 50 characters. Required.
+`triggers.severity` | Integer | Severity level for the trigger expressed as an integer: 5 = lowest; 4 = low; 3 = medium; 2 = high; 1 = highest. Trigger severity is part of the alert definition.
```

Comment:
This is a nit, but I would reorder the list so it starts with 1 and ends with 5.